pallets/creditcoin/src/tests.rs

```diff
@@ -6,8 +6,9 @@ use crate::{
 use bstr::B;
 use codec::{Decode, Encode};
 use ethereum_types::H256;
-use frame_support::{assert_noop, assert_ok, traits::Get, BoundedVec};
+use frame_support::{assert_noop, assert_ok, traits::Get, weights::{GetDispatchInfo, Pays}, BoundedVec};
 use frame_system::RawOrigin;
+use pallet_transaction_payment;
```

Comment:
@nathanwhit here I get

```
   |
11 | use pallet_transaction_payment;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^ no external crate `pallet_transaction_payment`
```

Not sure how to make use of this pallet. My idea was to call `compute_fee()` with the `call_info` value and assert the minimum value.